external/ilasm/ilasm.depproj

```diff
@@ -6,6 +6,22 @@
     <NugetRuntimeIdentifier>$(RuntimeOS)-x64</NugetRuntimeIdentifier>
     <OutputPath>$(ToolsDir)ilasm</OutputPath>
     <IsRuntimeAssembly>false</IsRuntimeAssembly>
+    <RidSpecificAssets>true</RidSpecificAssets>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Platforms">
+      <Version>2.0.0-$(CoreFxExpectedPrerelease)</Version>
```

Comment:
I imagine we will do a clean-up pass of a lot of these versions once we get fully moved to the msbuild based CLI.